app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsFormModal/LocalUnitsForm/index.tsx

```diff
@@ -299,6 +309,13 @@ function LocalUnitsForm(props: Props) {
         [validate, localUnitId, setError, updateLocalUnit, addLocalUnit],
     );
 
+    const handleDeleteLocalUnitClick = useCallback(
+        () => {
+            setShowDeleteLocalUnitModalTrue();
+        },
+        [setShowDeleteLocalUnitModalTrue],
+    );
```

Comment:
We don't need to define this function. We can just use setShowDeleteLocalUnitModalTrue